Cargo.toml

```diff
@@ -27,12 +27,16 @@ option_if_let_else = "warn"
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
+[patch.crates-io]
+alloy-rpc-types-engine = { git = "https://github.com/refcell/alloy", branch = "rf/engine/opt-serde" }
+
```

Comment:
TODO: need to remove this once https://github.com/alloy-rs/alloy/pull/1283 is merged.